ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java

```diff
@@ -223,6 +226,29 @@ public void setCollapsable(ReactViewGroup view, boolean collapsable) {
     // handled in NativeViewHierarchyOptimizer
   }
 
+  @ReactProp(name = "clickable")
+  public void setClickable(final ReactViewGroup view, boolean clickable) {
+    if (clickable) {
+      view.setOnClickListener(
+              new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                  final EventDispatcher mEventDispatcher = ((ReactContext)view.getContext()).getNativeModule(UIManagerModule.class)
+                          .getEventDispatcher();
+                  mEventDispatcher.dispatchEvent(new ViewGroupClickEvent(view.getId()));
+                }});
+
+      // Clickable elements are focusable. On API 26, this is taken care by setClickable.
+      // Explicitly calling setFocusable here for backward compatibility.
+      view.setFocusable(true /*isFocusable*/);
+    }
+    else {
+      view.setOnClickListener(null);
+      view.setClickable(false);
+      view.setFocusable(false);
```

Comment:
I think this may cause some problems. Just because something is set to not be clickable, doesn't mean it should explicitly be not focusable.  

For example, if you set accessible="true" (which sets view.setFocusable(true) and clickable="false", (which sets view.setFocusable(false), it would contradict here and I'm not sure which would actually win out.

Reply:
My initial thought was that if you enable click on a view with setClickable(true) and then setClickable(false) later, you would want it back to the way it was. But I agree that this could introduce unwanted behavior. I will remove `view.setFocusable(false)`, but we'll need to keep setting it to true alongside `setOnClickListener`